src/components/Accessibility/SkipLink.tsx

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 
 interface SkipLinkProperties {
-  containerRef: React.RefObject<HTMLElement>;
-  children: React.ReactElement;
+  containerRef: React.RefObject<HTMLElement | null>;
+  children: React.ReactElement<{
+    onClick: (e: React.SyntheticEvent) => unknown;
```

Comment:
Fixes an issue with the call to cloneElement below. Basically we need to pass the expected props of the element;